src/test/shell/bazel/bazel_workspace_status_test.sh

```diff
@@ -184,4 +184,32 @@ EOF
 
 }
 
+function test_env_var_in_workspace_status() {
+  create_new_workspace
+  cat >$TEST_TMPDIR/wsc.sh <<'EOF'
+#!/bin/bash
+
```

Comment:
add a non-stable key as well, assert that changing it will not trigger a rebuild